go/vt/vtgate/resilient_srv_topo_server.go

```diff
@@ -273,6 +278,23 @@ func (server *ResilientSrvTopoServer) GetSrvKeyspace(ctx context.Context, cell,
 		server.srvKeyspaceCache[key] = entry
 	}
 	server.mutex.Unlock()
+	return entry
+}
+
+// GetSrvKeyspace returns SrvKeyspace object for the given cell and keyspace.
+func (server *ResilientSrvTopoServer) GetSrvKeyspace(ctx context.Context, cell, keyspace string) (*topodatapb.SrvKeyspace, error) {
+	server.counts.Add(queryCategory, 1)
```

Comment:
I don't think we're interested in QPS on this any more, it's just querying an inside cached value... I would remove that counter.